packages/backend/src/routers/projectRouter.ts

```diff
@@ -260,7 +260,7 @@ projectRouter.get(
                 ? req.query.chartUuid.toString()
                 : undefined;
 
-        const includePrivate = req.query.includePrivate === 'true';
```

Comment:
that's because we have changed method `hasDirectAccessToSpace`, this method would stop the admin to access the private space that they don't have access to, unless `includePrivate` is explicitly set to true to skip this check.
so i invert the condition check here and when its undefined it won't exclude  the admin from private space access.